Fixed Spelling Mistake

bauripalash opened the pull request:
behavior (american english) > behaviour (Commonwealth English)

https://en.wikipedia.org/wiki/Behavior

and added 'architecture'

chrisbra commented:
What is architecture needed for and second, what is the reason to switch from american english to british english? We have probably a lot of american english in the documentation, so I am not fond of changing that.

bauripalash commented:
### why did i added architecture ?
maybe errors are occurring due to architecture of OS. though i know that's not so important

### why did i switched to British English
while opening a issue , my Firefox Web Browser was showing a red underline (a spelling error). I think this happened because my language is set to Indian English. then i googled about it i found the Wikipedia article.

### I know both of the changes are not so important  :smiley:

chrisbra commented:
thanks. Let's just keep it as is.
